README.md

```diff
@@ -108,6 +110,8 @@ terraform apply -var="installation_name=test" -var="management_cluster_oidc_prov
 ```
 export INSTALLATION_NAME=test
 export MANAGEMENT_CLUSTER_OIDC_PROVIDER_DOMAIN=irsa.test.gaws.gigantic.io
+# Optional: only set to true if this installation is going to be used exclusively to create WCs on existing infrastructure
```

Comment:
I find `existing infrastructure` to be a bit vague. Would it make sense to say `existing VPCs and subnets`?

Reply:
right, fixed